### ISSUE TITLE
Enhancement - Tickets by priority

### ISSUE THREAD

**Issue by PaulSells** [campai-bv/autotask-dashboards-os#29]
Would be useful to have a chart showing a summary of tickets by priority

**Comment by coencoppens** [comment]
@PaulSells What would you like that chart to look like? I imagine it wouldn't be a chart, but a table like the top X tickets is now only with different sorting.

**Comment by PaulSells** [comment]
@coencoppens Yes or possibly like the queue health showing total tickets by priority and then two further columns showing avg days open and overdue
